pkgs/top-level/all-packages.nix

```diff
@@ -24870,6 +24870,10 @@ with pkgs;
 
   tet = callPackage ../development/tools/misc/tet { };
 
+  texpresso = callPackage ../tools/typesetting/tex/texpresso {
+    tectonic-unwrapped = callPackage ../tools/typesetting/tex/texpresso/tectonic.nix { };
```

Comment:
Is there a reason to call this tectonic-unwrapped, when it seems to be a forked version of tectonic?

Reply:
I know, I'm just not sure if that's a good idea, as it represents something different

Reply:
What would you propose?

Reply:
Something like tectonic-texpresso?

Reply:
Renamed to `texpresso-tectonic`